### ISSUE TITLE
Ceph Cluster

### ISSUE THREAD

**Issue by MJP-76** [minkebox/pimox#1]
Just wondering if you have tried or if possible to run a Ceph Cluster

Idea is to run 3 x PI 4's 8GB all with USB3.0 256GB SSD as a Ceph cluster and run a few containers for home automation like Home Assistant, MQTT, CCTV etc

If not a cluster running them anyway would be awesome

Will test and feedback

**Comment by mlch911** [comment]
Have you tried this, Bro ?

**Comment by aanon4** [comment]
Yes I run a Ceph cluster like that (512GB SSDs but otherwise as you describe). I run a few VMs on top of it.

**Comment by mlch911** [comment]
> Yes I run a Ceph cluster like that (512GB SSDs but otherwise as you describe). I run a few VMs on top of it.

Thanks a lot.
